feature-configs/base/sriov/05-sriov-network.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: sriov-network
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |
```

Comment:
Can we kustomize this section? So we'll provide the relevant ipam depending on the environment (in demo we might have dhcp)? Wdyt?